src/main/java/com/ust/sourcecourse/configuration/controller/DBDataSourceController.java

```diff
@@ -42,4 +46,40 @@ public ResponseEntity<List<DBDataSourceInfo>> getDBSourceInfo() {
 		return ResponseEntity.ok(dataSourceService.getDBInfo());
 	}
 
+	@GetMapping("/tags/columns/{uid}")
+	public ResponseEntity<List<String>> getColumnTags(@PathVariable Long uid) {
+		List<String> columnTags = dataSourceService.getColumnTags(uid);
+		return ResponseEntity.ok(columnTags);
+	}
+
+	@GetMapping("/tags/tables/{uid}")
+	public ResponseEntity<List<String>> getTableTags(@PathVariable Long uid) {
+		List<String> tableTags = dataSourceService.getTableTags(uid);
+		return ResponseEntity.ok(tableTags);
+	}
+
+	@PutMapping("/tags/columns/{uid}")
```

Comment:
Since we are updating tags and description, url need not have "tags" path. Pls remove

Reply:
resolved